applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -31,6 +31,18 @@ namespace Python {
 
 void AddCustomUtilitiesToPython(py::module& m)
 {
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, COORDINATES)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, TANGENTS)
+
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, CROSS_AREA)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, PRESTRESS_CAUCHY)
+
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, SHAPE_FUNCTION_VALUES)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, SHAPE_FUNCTION_LOCAL_DERIVATIVES)
+
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RAYLEIGH_ALPHA)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RAYLEIGH_BETA)
```

Comment:
Are they really needed here?

Reply:
I think that they are used inside your base classes.

Can you apply the proposed changes to the base classes?